Docker/Compose/pgsql/env.example

```diff
@@ -1,6 +1,6 @@
 API_MODE=SharedInstance
 ADMIN_API_VIRTUAL_NAME=adminapi
-ODS_VIRTUAL_NAME=webapi
+ODS_API_VERSION=6.1
 
```

Comment:
@simpat-adam 
Please provide the description instead of actual value ODS_API_VERSION=<Version number of ODS API eg. 5.3, 5.3-cqe, 6.0, 6.1> on the .env.example files